README.yaml

```diff
@@ -28,9 +28,21 @@ include:
   - "docs/targets.md"
 
 usage: |-
-  1. Run `make init`
-  2. Run `COMPONENTS_BUILD=true/false make run` to start local server
-  3. Run `make build` to generate static site in `public/`
+  1. Download [Docker](https://www.docker.com/).
+  2. Install the [Atom editor](https://atom.io/)(optional).
+
```

Comment:
Try playing with the spacing here. The formatting is still wrong. Bullets should be indented.

![image](https://user-images.githubusercontent.com/52489/50788270-c0639e00-126d-11e9-9747-509659703195.png)